src/Controls/src/Xaml/MarkupExtensions/AppThemeBindingExtension.cs

```diff
@@ -60,6 +60,8 @@ BindingBase IMarkupExtension<BindingBase>.ProvideValue(IServiceProvider serviceP
 			else
 				bp = valueProvider.TargetProperty as BindableProperty;
 
+			propertyType = bp?.ReturnType ?? throw new InvalidOperationException("Cannot determine property to provide the value for.");
+
```

Comment:
This was the old code:

```cs
propertyType = bp?.ReturnType
							  ?? pi?.PropertyType
							  ?? throw new InvalidOperationException("Cannot determine property to provide the value for.");
```

Reply:
Right, but here the `pi` variable was already removed so I removed it from this expression. I honestly don't even know what this code does, but it seems that the PR that changed it made it so that the code:

1. Has some unused code paths now (some variables are always null)
2. At least on my machine, causes MAUI apps to crash 100% of the time

I think (1) is something that is clearly incorrect in the code. I'm not sure how bad (2) is outside of my machine, but it definitely affects my machine (I'm a bit surprised others haven't run into this??).